peps/pep-9999.rst

```diff
@@ -995,109 +995,62 @@ of the impact of the operation should be provided.
 How to Teach This
 =================
 
-There are at least four audiences that need to learn a workflow here.
+There are at least four audiences that may need to get familiar with the contents of this PEP:
 
-1. Python package maintainers wishing to express an external dependency.
+1. Central registry maintainers, who are responsible for curating the list of
+   well-known DepURLs and mapped ecosystems.
 2. Packaging ecosystem maintainers, who are responsible for keeping the
-   mapping for their ecosystem current.
-3. Core registry maintainers, who are responsible for curating the central
-   repository of DepURLs and their descriptions.
+   mapping for their ecosystem up-to-date.
+3. Maintainers of Python projects that require external dependencies.
 4. End users of packages that have external dependency metadata.
```

Comment:
Do we want to add a fifth case, "Package builders"? We could discuss how they'd need to get familiar with the tooling that facilitates the automated installation of external dependencies. Maybe as part of (4)?